hooks/update_config.js

```diff
@@ -21,18 +21,22 @@ module.exports = function(context) {
     var xwalkVariables = {},
         argumentsString = context.cmdLine,
         pluginConfigurationFile = path.join(context.opts.plugin.dir, 'plugin.xml'),
-        androidPlatformDir = path.join(context.opts.projectRoot,
-            'platforms', 'android'),
-        projectConfigurationFile = path.join(context.opts.projectRoot,
-            'config.xml'),
-        platformConfigurationFile = path.join(androidPlatformDir,
-            'res', 'xml', 'config.xml'),
-        projectManifestFile = path.join(androidPlatformDir,
-            'AndroidManifest.xml'),
+        androidPlatformDir = path.join(context.opts.projectRoot, 'platforms', 'android'),
```

Comment:
`androidPlatformDir`, `projectConfigurationFile` and `projectManifestFile` have the same values they did before, so could you revert the style changes?

Reply:
Done.